src/component/mxgraph/StyleConfigurator.ts

```diff
@@ -110,27 +108,13 @@ export default class StyleConfigurator {
     });
   }
 
-  private configureServiceTaskStyle(): void {
-    const style = this.mxUtils.clone(this.getStylesheet().getCellStyle(ShapeBpmnElementKind.TASK_USER), this.getDefaultVertexStyle());
-    style[this.mxConstants.STYLE_STROKECOLOR] = 'red';
-    this.putCellStyle(ShapeBpmnElementKind.TASK_SERVICE, style);
-  }
-
-  private configureUserTaskStyle(): void {
-    const style = this.cloneDefaultVertexStyle();
-    style[this.mxConstants.STYLE_SHAPE] = this.mxConstants.SHAPE_RECTANGLE;
-    style[this.mxConstants.STYLE_VERTICAL_ALIGN] = 'middle';
-    style[this.mxConstants.STYLE_STROKECOLOR] = '#2C6DA3';
-    style[this.mxConstants.STYLE_STROKEWIDTH] = 2;
-    style[this.mxConstants.STYLE_ROUNDED] = true;
-    this.putCellStyle(ShapeBpmnElementKind.TASK_USER, style);
-  }
-
-  private configureTaskStyle(): void {
-    const style = this.cloneDefaultVertexStyle();
-    style[this.mxConstants.STYLE_SHAPE] = ShapeBpmnElementKind.TASK;
-    style[this.mxConstants.STYLE_VERTICAL_ALIGN] = 'middle';
-    this.putCellStyle(ShapeBpmnElementKind.TASK, style);
+  private configureTasksStyle(): void {
+    ShapeUtil.taskKinds().forEach(kind => {
+      const style = this.cloneDefaultVertexStyle();
+      style[this.mxConstants.STYLE_SHAPE] = kind;
+      style[this.mxConstants.STYLE_VERTICAL_ALIGN] = 'middle';
```

Comment:
Don't we need to preserve here the perimeter?
(like we did for Event and Gateway: style[mxConstants.STYLE_PERIMETER] = mxPerimeter.RectanglePerimeter;)

Reply:
yes good catch
can you handle it please @aibcmars ?

Reply:
apparently @csouchet confirmed it is not needed for shapes that extend RectangleShape